src/windows/GrammarWindow.java

```diff
@@ -0,0 +1,120 @@
+/*
+ * To change this license header, choose License Headers in Project Properties.
+ * To change this template file, choose Tools | Templates
+ * and open the template in the editor.
+ */
+package windows;
+
+import inka.Inka;
+import java.awt.BorderLayout;
+import java.awt.Color;
+import java.awt.Container;
+import java.awt.FlowLayout;
+import java.awt.GridLayout;
+import java.awt.event.ActionEvent;
+import java.awt.event.ActionListener;
+import javax.swing.JButton;
+import javax.swing.JFrame;
+import javax.swing.JLabel;
+import javax.swing.JPanel;
+import javax.swing.border.LineBorder;
+
+/**
+ *
+ * @author me
+ */
+public class GrammarWindow extends JFrame {
+    
+    JPanel mainPanel = new JPanel();
+    private JPanel grammar = new JPanel();
+    private JPanel menuPanel = new JPanel();
+    private JPanel wordPanel = new JPanel();
+    private static JLabel jlabel = new JLabel();
+    
+    //private MainWindow mainWindow;
+        
+    
+    public GrammarWindow() {
+        this.mainPanel.setLayout(new GridLayout());
+        this.grammar.setLayout(new BorderLayout());
+        this.menuPanel.setLayout(new FlowLayout());
+        this.wordPanel.setLayout(new FlowLayout());
+        
+        mainPanel.setBorder(new LineBorder(Color.BLACK));
+        grammar.setBorder(new LineBorder(Color.BLACK));
+        menuPanel.setBorder(new LineBorder(Color.BLACK));
+        wordPanel.setBorder(new LineBorder(Color.BLACK));
+        
+        createGrammarWindow();
+    }
+    
+    
+    
+    private void createGrammarWindow() {        
+        addButton("Menu", menuPanel, 0);
+        addButton("Ask", menuPanel, 0);
+        addButton("Card manager", menuPanel, 0);
+        
+        jlabel.setText("Example sentence, which is ...");
+        grammar.add(jlabel);
+        
+        addButton("bad", wordPanel, 1);
+        addButton("wrong", wordPanel, 1);
+        addButton("good", wordPanel, 1);
+        
+        grammar.add(menuPanel, BorderLayout.NORTH);
+        grammar.add(wordPanel, BorderLayout.SOUTH);
+        mainPanel.add(grammar);
+    }
+    
+    //Get text to set button text, container, where to add the button and integer to which listener call
+    private static void addButton(String text, Container container, int whichListener){
```

Comment:
I would write `private static void addButton(String text, Container container, ActionListener actionListener) { ... }`. This way, any kind of object that was inherited from ActionListener interface could be passed as an argument, and then you could write
```
           button.addActionListener(actionListener);
           container.add(button);
```
The only drawback of it is that you have to instantiate an ActionListener before you call the function. But this way it becomes easier to read and you can set other properties of the passed ActionListener outside of the function. This is what I call 'more abstract'. :)